DataProcessing/data_tagger.py

```diff
@@ -0,0 +1,102 @@
+import os
+import pickle
+from collections import defaultdict
+from matplotlib import pyplot as plt
+
+
+# todo all of these actions need to be done to the db, currently they all run in O(n) for each Crop changed !
+from DataProcessing.dataProcessingConstants import ID_TO_NAME
+
+
+def mark_vague(db, track, crop_inds):
+    for crop_id in crop_inds:
+        db_index = db.index(track[crop_id])
+        db[db_index].is_vague = True
+
+def relabel_all(db, track, new_label):
+    for crop in track:
+        db_index = db.index(crop)
+        db[db_index].set_label(new_label)
+
+def discard_crops(db, track, crop_inds):
+    # todo this should be done later to keep presistance of ids, or at the end of the track
+    for crop_id in crop_inds:
+        db_index = db.index(track[crop_id])
+        del db[db_index] # todo should we del the actual crops as well?
+
+def split_track(db, track, split_start, split_end , new_label):
+    splitted_track = track[split_start:split_end]
+    new_track_id = max(crop.track_id for crop in db) + 1
+    for crop in splitted_track:
+        db_index = db.index(crop)
+        db[db_index].track_id = new_track_id
+        db[db_index].set_label(new_label)
+
+def insert_new_label():
+    print("Please insert of of the following Ids:")
+    print(ID_TO_NAME)
+    new_label_name = ID_TO_NAME[int(input())]
+    return new_label_name
+
+
+def label_tracklets(crop_db_path:str):
+    def _label_tracklet(crops_pkl, track:list):
+        """
+
+        Args:
+            track: A list of Crop objects representing the track
+        Returns:
+        """
+        # todo deal with face images
+        NUM_OF_CROPS_TO_VIEW = 25
+        counter = 0
+        for batch in range(0,len(track),NUM_OF_CROPS_TO_VIEW):
+            cur_batch = track[batch:min(batch+NUM_OF_CROPS_TO_VIEW, len(track))]
+            _, axes = plt.subplots(5, 5, figsize=(10,10))
+            axes = axes.flatten()
+            for a in axes:
+                a.axis('off')
+            # axes = [a.axis('off') for a in axes]
+            for crop,ax in zip(cur_batch, axes):
+                # using / on to adapt to windows env
+                img_path = os.path.join(crop_db_path + '//' + crop.unique_crop_name + '.png')
```

Comment:
This will cause problems when running on my mac or on the server.
Try the following suggestion on your computer:
```suggestion
                img_path = os.path.join(crop_db_path + os.sep, crop.unique_crop_name, '.png')
```
`os.sep` should be '\' when you use it in windows and `/` for linux, this way it will probably work for both you computer and the server.